nautobot/ipam/tables.py

```diff
@@ -255,13 +255,18 @@ class VRFDeviceAssignmentTable(BaseTable):
         accessor="virtual_machine.name",
         verbose_name="Virtual Machine",
     )
+    virtual_device_context = tables.Column(
```

Comment:
Not critical, but I wonder if it'd be better UX to have two columns `related object type` and `related object name` instead of three columns `device`/`virtual_machine`/`virtual_device_context` that are mutually exclusive in their data?

Reply:
Added in https://github.com/nautobot/nautobot/pull/6936/commits/d5efbd8ff23602a85a79b73ae0edbe57e30860f1
![Screenshot 2025-02-25 at 2 25 27 PM](https://github.com/user-attachments/assets/c7b47458-a404-48ef-897f-5e45a43fdb82)